db/migrations/20190616213359-add-status-to-products.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn(
+      'products',
+      'dataInputStatus',
+      Sequelize.TEXT,
+      {
+        defaultValue: 'incomplete'
```

Comment:
you can leave this as is, no big deal, but next time if it's short just keep it 1-line: 

```
{ defaultValue: 'incomplete' },
```

oh, also, I like to always put a trailing comma at the end of a multi-line object or array, makes it easier to add new items in the future, but it's just a preference, it's fine the way it is